### DISCUSSION TITLE
Budgie 10.7: Superhero Raven

### DISCUSSION THREAD

**Discussion by JoshStrobl** [BuddiesOfBudgie/budgie-desktop#279] [Announcements]
![FeaturedImage](https://user-images.githubusercontent.com/156574/215338911-011c0c9a-8449-47c1-b913-705103604698.jpg)

We are excited to announce the release of Budgie Desktop 10.7 🎉  Budgie 10.7 is a brand new release series for Budgie Desktop, featuring major re-architectures, new APIs for extensibility, and polish to the user experience.

Learn more at https://blog.buddiesofbudgie.org/budgie-10-7-released/

<hr /><em>This discussion was created from the release <a href='https://github.com/BuddiesOfBudgie/budgie-desktop/releases/tag/v10.7'>Budgie 10.7: Superhero Raven</a>.</em>

**Comment by ViktorOn** [comment]
Hi. Can someone explain to me why it is not possible to round windows both at the top and bottom?

**Comment by OlivierDuchateau** [comment]
I've just submitted [review](https://reviews.freebsd.org/D37224?id=116528#872810) for FreeBSD, in order to have Budgie available on this system too.
